vm/shell.cpp

```diff
@@ -233,15 +233,17 @@ static cell_t Copy2dArrayToCallback(IPluginContext* cx, const cell_t* params)
 
   cell_t addr;
   if (!cx->HeapAlloc2dArray(params[2], params[3], &addr, flat_array))
-    return 0;
+    return cx->ThrowNativeError("Could not allocate %d * %d 2D Array: %d", params[2], params[3], err);
```

Comment:
Functions returning "err" have already called ReportError, and don't have an error code.

Reply:
Sorry I forgot to remove the left messages